backend/src/packages/auth/auth.service.ts

```diff
@@ -75,8 +75,11 @@ class AuthService {
     };
   }
 
-  public getAuthenticatedUser(id: number): Promise<UserAuthResponseDto | null> {
-    return this.userService.findById(id);
+  public async getAuthenticatedUser(id: number): Promise<UserAuthResponseDto> {
+    const user = (await this.userService.findById(id)) as UserAuthResponseDto;
```

Comment:
we need to add modificator to the getById query not to return password from db

Reply:
got it, will change it